### PR TITLE
chore: [main/1.2] wrong paths to plugins in supported plugins table

### PR DESCRIPTION
**Pull request by nickboldt**
### What does this PR do?

chore: wrong paths to plugins in supported plugins table

Signed-off-by: Nick Boldt <nboldt@redhat.com>

### Screenshot/screencast of this PR
N/A

### What issues does this PR fix or reference?
N/A (or see commit message above for issue number)

### How to test this PR?
N/A

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)

- [x] [The Eclipse Contributor Agreement is valid](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-eclipse-contributor-agreement-is-valid)
- [x] [Code produced is complete](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-produced-is-complete)
- [x] [Code builds without errors](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-builds-without-errors)
- [ ] [Tests are covering the bugfix](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#tests-are-covering-the-bugfix)
- [x] [The repository devfile is up to date and works, if one exists](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-repository-devfile-is-up-to-date-and-works)
- [x] [Sections What issues does this PR fix or reference and How to test this PR completed](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#sections-what-issues-does-this-pr-fix-or-reference-and-how-to-test-this-pr-completed)
- [x] [Relevant user documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [x] [Relevant contributing documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [x] [CI/CD changes implemented, documented and communicated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#cicd-changes-implemented-documented-and-communicated)

### Reviewers

Reviewers, please comment how you tested the PR when approving it.

### CODE REVIEW

**Comment by nickboldt** [comment]
note: this needs to be cherrypicked to 1.1.x docs, AND is not complete. some paths may be missing the `-dynamic` suffix. cc: @davidfestal

File: modules/rhdh-plugins-reference/ref-rhdh-plugins.adoc [hunk 1]
```diff
@@ -111,15 +111,15 @@ requests, etc. which is used by the Azure DevOps frontend plugin. |0.5.5
...
+|Bitbucket |Backend |@backstage/plugin-catalog-backend-module-bitbucket-cloud |A Backstage catalog backend module that helps integrate towards Bitbucket Cloud. |0.1.28 |Community Support |./dynamic-plugins/dist/backstage-plugin-catalog-backend-module-bitbucket-cloud |`BITBUCKET_WORKSPACE` |Disabled
 
-|Bitbucket |Backend |@backstage/plugin-catalog-backend-module-bitbucket-server |A Backstage catalog backend module that helps integrate towards Bitbucket Server. |0.1.26 |Community Support |dynamic-plugins/wrappers/backstage-plugin-catalog-backend-module-bitbucket-server |`BITBUCKET_HOST` |Disabled
+|Bitbucket |Backend |@backstage/plugin-catalog-backend-module-bitbucket-server |A Backstage catalog backend module that helps integrate towards Bitbucket Server. |0.1.26 |Community Support |./dynamic-plugins/dist/backstage-plugin-catalog-backend-module-bitbucket-server |`BITBUCKET_HOST` |Disabled
```
Comment (davidfestal) [inline modules/rhdh-plugins-reference/ref-rhdh-plugins.adoc:118]:
```suggestion
|Bitbucket |Backend |@backstage/plugin-catalog-backend-module-bitbucket-server |A Backstage catalog backend module that helps integrate towards Bitbucket Server. |0.1.26 |Community Support |./dynamic-plugins/dist/backstage-plugin-catalog-backend-module-bitbucket-server-dynamic |`BITBUCKET_HOST` |Disabled
```

File: modules/rhdh-plugins-reference/ref-rhdh-plugins.adoc [hunk 2]
```diff
@@ -111,15 +111,15 @@ requests, etc. which is used by the Azure DevOps frontend plugin. |0.5.5
...
 
-|Bitbucket |Backend |@backstage/plugin-scaffolder-backend-module-bitbucket-cloud |The Bitbucket Cloud module for @backstage/plugin-scaffolder-backend |0.1.3 |Community Support |dynamic-plugins/wrappers/backstage-plugin-scaffolder-backend-module-bitbucket-cloud | |Enabled
+|Bitbucket |Backend |@backstage/plugin-scaffolder-backend-module-bitbucket-cloud |The Bitbucket Cloud module for @backstage/plugin-scaffolder-backend |0.1.3 |Community Support |./dynamic-plugins/dist/backstage-plugin-scaffolder-backend-module-bitbucket-cloud | |Enabled
```
Comment (davidfestal) [inline modules/rhdh-plugins-reference/ref-rhdh-plugins.adoc:120]:
```suggestion
|Bitbucket |Backend |@backstage/plugin-scaffolder-backend-module-bitbucket-cloud |The Bitbucket Cloud module for @backstage/plugin-scaffolder-backend |0.1.3 |Community Support |./dynamic-plugins/dist/backstage-plugin-scaffolder-backend-module-bitbucket-cloud-dynamic | |Enabled
```

File: modules/rhdh-plugins-reference/ref-rhdh-plugins.adoc [hunk 3]
```diff
@@ -111,15 +111,15 @@ requests, etc. which is used by the Azure DevOps frontend plugin. |0.5.5
...
 
-|Bitbucket |Backend |@backstage/plugin-scaffolder-backend-module-bitbucket-server |The Bitbucket Server module for @backstage/plugin-scaffolder-backend. |0.1.3 |Community Support |dynamic-plugins/wrappers/backstage-plugin-scaffolder-backend-module-bitbucket-server | |Enabled
+|Bitbucket |Backend |@backstage/plugin-scaffolder-backend-module-bitbucket-server |The Bitbucket Server module for @backstage/plugin-scaffolder-backend. |0.1.3 |Community Support |./dynamic-plugins/dist/backstage-plugin-scaffolder-backend-module-bitbucket-server | |Enabled
```
Comment (davidfestal) [inline modules/rhdh-plugins-reference/ref-rhdh-plugins.adoc:122]:
```suggestion
|Bitbucket |Backend |@backstage/plugin-scaffolder-backend-module-bitbucket-server |The Bitbucket Server module for @backstage/plugin-scaffolder-backend. |0.1.3 |Community Support |./dynamic-plugins/dist/backstage-plugin-scaffolder-backend-module-bitbucket-server-dynamic | |Enabled
```

**Comment by nickboldt** [comment]
squashed and rebased; see also https://github.com/redhat-developer/red-hat-developers-documentation-rhdh/pull/155 for 1.1.x branch.

**Review by davidfestal — APPROVED** [review]

**Pull request merged**